Python/ceval.c

```diff
@@ -4722,6 +4722,7 @@ _PyEval_EvalFrameDefault(PyThreadState *tstate, InterpreterFrame *frame, int thr
             Py_DECREF(obj);
             STACK_SHRINK(call_shape.postcall_shrink);
             SET_TOP(res);
+            CHECK_EVAL_BREAKER();
```

Comment:
This, and several of the other `CHECK_EVAL_BREAKER` additions are not needed.
See me review comments on https://github.com/python/cpython/pull/30826

Reply:
It looks like those comments have been buried, I'll note the cases here.